content/tokio/tutorial/spawning.md

```diff
@@ -255,9 +255,9 @@ before its type is no longer valid.
 
 When we say that a value is `'static`, all that means is that it would not be
 incorrect to keep that value around forever. This is important because the
-compiler is unable to reason about how long a newly spawned task stays around,
-so the only way it can be sure that the task doesn't live too long is to make
-sure it may live forever.
+compiler is unable to reason about how long a newly spawned task stays around.
+We have to make sure the task is allowed to live forever, in order to let the compiler
+give the permission to tokio to make the task run as long as it needs to.
```

Comment:
```suggestion
We have to make sure that the task is allowed to live forever, so that Tokio can make the task run as long as it needs to.
```

Reply:
That's more concise ! It does not mention the compiler needing to check the lifetime of the task, but since it's mentioned above, I think it's clear like that :)